src/test/java/com/epam/mjc/io/FileReaderTest.java

```diff
@@ -0,0 +1,99 @@
+package com.epam.mjc.io;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class FileReaderTest {
+
+    private final static Path TEST_FILE = Path.of("src/main/resources/Profile.txt");
+    private final static Path READER_CLASS = Path.of("src/main/java/com/epam/mjc/io/FileReader.java");
+    private static final String PROFILE_PARAM_VALUE_DELIMETER = ": ";
+    private static final String SOURCE_CODE_LOCATION = "src/main/java";
+
+
+    @Test
+    public void testProgramReadsDataCorrectly () {
+        FileReader fileReader = new FileReader();
+        Profile actual = fileReader.getDataFromFile(new File(TEST_FILE.toUri()));
+        Profile expected = readFile(TEST_FILE);
```

Comment:
Sorry, but student can look into this file and copy logic from here.

Reply:
Any ideas?

Reply:
You can create Profile:
`Profile expected = new Profile(name, phone...);`
And I suggest duplicate `.txt` file and move one in `test/resources` or create another file in this directory.

Reply:
Simple, but effective. I agree

Reply:
Fixed